src/components/sections/LocationCardFactory.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom'; // Import Link from React Router
+import Parse from 'parse/dist/parse.min.js';
+import { List } from 'antd';
+import Card from 'react-bootstrap/Card';
+import Container from 'react-bootstrap/Container';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import '../../styles/index.css';
+
+export const LocationCardFactory = () => {
+    const [readResults, setReadLocations] = useState([]);
```

Comment:
why is the state var named different then the setter?

Reply:
Good point, we updated it to readLocations